submit_cgap/submission.py

```diff
@@ -825,7 +844,12 @@ def upload_file_to_new_uuid(filename, schema_name, auth, **context_attributes):
 
     post_item = compute_file_post_data(filename=filename, context_attributes=context_attributes)
 
-    response = ff_utils.post_metadata(post_item=post_item, schema_name=schema_name, key=auth)
+    if DEBUG_PROTOCOL:  # pragma: no cover
+        show("Creating FileOther type object ...")
+    response = portal_metadata_post(schema=schema_name, data=post_item, auth=auth)
+    if DEBUG_PROTOCOL:  # pragma: no cover
+        show(f"Created FileOther type object:"
+             f" {response.get('@graph', [{'uuid': 'not-found'}])[0].get('uuid', 'not-found')}")
```

Comment:
Almost the entirety of this string is a non-string. Maybe factor out as a local variable assignment and just refer to the local variable here?